.github/workflows/cargo-deny.yml

```diff
@@ -0,0 +1,42 @@
+name: cargo deny checks
+
+on:
+  pull_request:
+    paths:
+      - 'Cargo.lock'
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  cargo-deny:
+    strategy:
+      matrix: >-
+        ${{
+          fromJSON(
+            github.event_name == 'schedule'
+              && '{"ref":["main","release","release-proxy","release-compute"]}'
+              || format('{{"ref":["{0}"]}}', github.sha)
```

Comment:
I suppose we run it on release branches to understand if we have vulnerable code on prod (or we're going to deploy it soon)?
I'd probably run it more frequently (each 2-3 hours maybe) on `release*` branches. But that might be noisy, so I don't have a strong opinion about it

Reply:
The problem I see with that is that we're going to have 32 pings (or 48 if we go for 2h instead of 3h) to oncall-devprod per day until the fix is merged into main and release branches. I think that would make the cicd channel unusable for other stuff. Running that once per day should really be enough. If stuff is extremely urgent and short notice, I'm sure we'll notice in other ways as well.